xarray/core/variable.py

```diff
@@ -360,8 +360,43 @@ def compute(self, **kwargs):
         --------
         dask.array.compute
         """
-        new = self.copy(deep=False)
-        return new.load(**kwargs)
+        import dask
+        (result,) = dask.compute(self, **kwargs)
+        return result
+
+    def visualize(self, **kwargs):
+        import dask
+        return dask.visualize(self, **kwargs)
```

Comment:
My inclination would be to leave this out and require using `dask.visualize()`. My concern is that it could be easily confused with `.plot()`.

Reply:
Removed